chat-plugins/scavengers.js

```diff
@@ -928,7 +928,7 @@ let commands = {
 		points = parseInt(points);
 
 		if (!targetId || targetId === 'constructor' || targetId.length > 18) return this.errorReply("Invalid username.");
-		if (!points || points < 0) return this.errorReply("Points must be an integer greater than 0.");
+		if (!points || points < 0 || points === Infinity) return this.errorReply("Points must be a non-Infinity integer greater than 0.");
```

Comment:
Also if you actually care about the value being a real integer, you might want to cap it at the point where doubles lose integer precision (2^53 i.e. 9,007,199,254,740,992)